modules/perception/lidar/lib/detection/lidar_point_pillars/point_pillars_detection.cc

```diff
@@ -78,8 +78,8 @@ bool PointPillarsDetection::Detect(const DetectionOptions& options,
   Timer timer;
 
   int num_points;
-//  cur_cloud_ptr_ = std::make_shared<base::PointFCloud>(*original_cloud_);
-  cur_cloud_ptr_ = original_cloud_;  // TODO(chenjiahao): for emergency use
+  cur_cloud_ptr_ = std::shared_ptr<base::PointFCloud>(
+      new base::PointFCloud(*original_cloud_));
```

Comment:
Any reason for using `std::shared_ptr<T>(new T(...))` instead of `std::make_shared<T>(...)` ?

Reply:
Maybe because of https://github.com/ApolloAuto/apollo/pull/11564#issue-440748538

Reply:
Yes, the reason is as @xiaoxq commented.